src/services/helpers.ts

```diff
@@ -117,3 +119,32 @@ export function addPagersFn<T>(
     next,
   };
 }
+
+export function setCustomKeysConfigurationInRequestConfig(
+  httpInstance: HttpInstance,
+  requestOptions?: HttpRequestConfig
+) {
+  const customRequestKeys =
+    requestOptions?.customRequestKeys || requestOptions?.customKeys;
```

Comment:
(question) Is it not a possibility that we want to extend the customKeys with customRequestKeys? The same with the response, rather than redeclaring keys? Maybe it makes it too complex to deduce what your set of keys is?

Reply:
That is a design choice, both are just as easy to implement and both have advantages. 

If I see this, I would expect the `customKeys` are overwritten by the `customRequestKeys`. That is why I went with the overwrite. 
```
{
    customKeys: ['b`],
    customRequestKeys: ['a']
}
```

Reply:
Just a thought and maybe any future questions can be solved with a little comment over `setCustomKeysConfigurationInRequestConfig` ?